frontend/src/index.js

```diff
@@ -3,8 +3,6 @@ import ReactDOM from "react-dom";
 import App from "./App";
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <App />,
```

Comment:
For those that will be wondering:
https://stackoverflow.com/questions/61220424/material-ui-drawer-finddomnode-is-deprecated-in-strictmode

Reply:
![image](https://user-images.githubusercontent.com/37108355/116786878-b5ba1c00-aaa9-11eb-8115-b450dade1023.png)

Reply:
If you say so